wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ParticipantsStringFormatter.swift

```diff
@@ -119,18 +107,29 @@ final class ParticipantsStringFormatter {
         return [.link: ParticipantsCellViewModel.showMoreLinkURL]
     }
 
-    init(message: ZMConversationMessage, font: UIFont = .mediumFont, largeFont: UIFont = .largeSemiboldFont, textColor: UIColor = .from(scheme: .textForeground)) {
+    init(
+        message: ZMConversationMessage,
+        font: UIFont = .mediumFont,
+        largeFont: UIFont = .largeSemiboldFont,
+        textColor: UIColor = .from(
+            scheme: .textForeground
+        )
```

Comment:
💅 Having one argument could be a one liner? as you like

Reply:
Done!